### PR TITLE
Update some strings

### PR DESCRIPTION
**Pull request by dkanada**

### CODE REVIEW

**Comment by Artiume** [comment]
Think this'll impact weblate?

**Comment by JustAMan** [comment]
@dkanada how can this be fixing #804 if all I see is a lot of removed messages?..

**Comment by dkanada** [comment]
The `update` part of the title involves adding a missing string to the source language and removing any strings that don't exist in the source language.

**Comment by JustAMan** [comment]
> The `update` part of the title involves adding a missing string to the source language

@dkanada 
But I don't see any additions to `en-us.json`?..

**Comment by dkanada** [comment]
@Artiume I added a basic restart warning to the UPnP section as well, I didn't want to make drastic changes for the generic restart message yet.

**Comment by Artiume** [comment]
Yeah, that can be a project for another day. You want me to cancel my other PR? :P

File: src/strings/en-us.json [hunk 1]
```diff
@@ -1223,6 +1223,7 @@
     "PlayFromBeginning": "Play from beginning",
     "PlayNext": "Play next",
     "PlayNextEpisodeAutomatically": "Play next episode automatically",
+    "PlaybackErrorNoCompatibleStream": "There are no available formats that are compatible with this device.",
```
Comment (JustAMan) [inline src/strings/en-us.json]: Probably suggest a user to look for errors converting? As "no streams" should never actually be seen, it should either be "transcoding required but disabled" or "transcoding failed"...

File: src/components/libraryoptionseditor/libraryoptionseditor.js [hunk 2]
```diff
@@ -107,7 +107,7 @@ define(["globalize", "dom", "emby-checkbox", "emby-select", "emby-input"], funct
         if (!plugins.length) return html;
 
         html += '<div class="metadataFetcher" data-type="' + availableTypeOptions.Type + '">';
-        html += '<h3 class="checkboxListLabel">' + globalize.translate("LabelTypeMetadataDownloaders", availableTypeOptions.Type) + "</h3>";
+        html += '<h3 class="checkboxListLabel">' + globalize.translate("LabelTypeMetadataDownloaders", globalize.translate(availableTypeOptions.Type)) + "</h3>";
```
Comment (thornbill) [inline src/components/libraryoptionseditor/libraryoptionseditor.js]: This does not look right...

Comment (thornbill) [reply src/components/libraryoptionseditor/libraryoptionseditor.js]: Okay nevermind I think this is fine, but it looks really odd to translate the fallback value also... do you also need a fallback for the translation of the fallback? :joy:

Comment (dkanada) [reply src/components/libraryoptionseditor/libraryoptionseditor.js]: You'd think that, but the type is also a translatable string, which was causing the issue for music libraries. Shows and movies were only one word so they never looked odd but music types were always untranslated.

**Pull request merged**